src/components/ControlPanel.jsx

```diff
@@ -208,7 +209,10 @@ class ControlPanel extends React.Component {
           <hr className="white-line" />
 
           <div>
-            <button className="button">{this.props.translate('infoBox.transcribeReverse')}</button>
+            {(this.props.frame === 0)
+              ? <button className="button" onClick={()=>{this.props.dispatch(changeFrame(1))}}>{this.props.translate('infoBox.transcribeReverse')}</button>
+              : <button className="button" onClick={()=>{this.props.dispatch(changeFrame(0))}}>{this.props.translate('infoBox.transcribeFront')}</button>
```

Comment:
Can we hide this button if there is not a reverse side? (`currentSubject.locations.length === 1`, or something like that)

Reply:
Can do, gimme a second. 👌